src/monitor-api-requests.ts

```diff
@@ -64,6 +64,12 @@ export async function deleteMonitor(monitorId: number, monitorsApiKey: string, m
   return response;
 }
 
+/**
+ * TODO - refactor handleMonitorsApiResponse into this module
+ * and use it here instead of throwing an error
+ *
+ * This is done because getExistingMonitors calls searchMonitors
+ */
```

Comment:
Nit: I wouldn't bother adding a TODO to the codebase -- instead I would either add a ticket in Jira, or accept that this is unlikely to be done any time soon. In my experience TODOs in code bases tend to hang out for a long time and just confuse the issue. They can also go out of date over time.

Reply:
Oh. My intention was to make this an issue in Github and tag it as `hacktoberfest`, `good first issue` in a few weeks. :)